examples/analytics.amp.html

```diff
@@ -218,6 +218,25 @@
 </script>
 </amp-analytics>
 
+<!-- Lotame tracking.
+  Please replace the value for "account" (12345)
+  with the correct Lotame Client ID.
+-->
+<amp-analytics type="lotame">
+<script type="application/json">
+{
+  "vars": {
+    "account": 12345
+  },
+  "extraUrlParams": {
```

Comment:
adding extraUrlParams is not needed here. That is a platform level thing and publishers are probably aware of them. Plus, you could add more meaningful examples to lotame product documentation, i any.